pl_examples/quantization/requirements.txt

```diff
@@ -0,0 +1,4 @@
+neural-compressor
+--find-links https://download.pytorch.org/whl/torch_stable.html
+torch>=1.9.0+cpu
```

Comment:
Does this run only a CPU ?

Reply:
yes, For now, INC focus on CPU and will support intel GPU in the future.